backend/app/api/knowledge_document_side_effects.py

```
@@ -0,0 +1,88 @@
+                summary_service.trigger_kb_summary(
+                    kb_id, user_id, user_name, force=False, clear_if_empty=True
+                )
+            )
+        finally:
+            loop.run_until_complete(loop.shutdown_asyncgens())
+            loop.close()
+
+    except Exception as e:
+        logger.error(
+            f"[KnowledgeAPI] Failed to update KB summary after deletion: "
+            f"kb_id={kb_id}, error={str(e)}",
```

Ruff (0.15.20)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)